examples/fastpm-compat/fastpm.lua

```diff
@@ -0,0 +1,40 @@
+-- parameter file
```

Comment:
It was not merged. I kept the file here as an example to run identical set up with FastPM.